packages/webapp/src/components/Filter/FilterButton/index.jsx

```diff
@@ -0,0 +1,27 @@
+import { FiFilter } from 'react-icons/fi';
+import PropTypes from 'prop-types';
+import styles from './styles.module.scss';
+import TextButton from '../../Form/Button/TextButton';
+import clsx from 'clsx';
+
+const FilterButton = ({ isFilterActive, onClick }) => {
+  return (
+    <div className={clsx([styles.filterButtonContainer, isFilterActive && styles.active])}>
+      <TextButton className={styles.filterButton} onClick={onClick}>
+        <FiFilter data-cy="tasks-filter" className={styles.filterButtonIcon} />
```

Comment:
NIT (just noting): this Cypress selector is for tests that we don't have anymore, and in any case the filter is not specific to tasks so it's incorrectly named